examples/bakdata/atm-fraud-detection/pipeline.yaml

```diff
@@ -56,6 +56,7 @@
       jmx:
         enabled: false
     debug: true
+  prefix: ""
```

Comment:
isn't this already covered by the default set for kubernetes-app?

Reply:
yes, I haven't removed all the prefix definitions on the pipeline file. Now it should be fix.